pom.xml

```diff
@@ -268,7 +267,7 @@ under the License.
       <dependency>
         <groupId>org.eclipse.jetty</groupId>
         <artifactId>jetty-webapp</artifactId>
-        <version>9.4.51.v20230217</version>
+        <version>9.4.53.v20231009</version>
```

Comment:
TODO for myself: we should move off EOLd Jetty, probably can lift some code from resolver tests that also did a smooth migration in test used "server" from 9.x to 10.x

Reply:
yep. the only reason I didn't change is because I know from running my old blog on a rapsi, that Jetty 10.x changed quite a few tings internally (module system etc and even some property names of the plugin) which can break logging for example. So I didn't want to blindly bump the major version without knowing the usages very well.